ServerCore/wwwroot/js/site.js

```diff
@@ -1 +1,6 @@
 ﻿// Write your Javascript code.
+$("time").each(function (elem) {
+    var utctimeval = $(this).html();
+    var date = new Date(utctimeval);
+    $(this).html(date.toLocaleString());
```

Comment:
Does it get translated to local time when it gets put into a Date object? I think toLoacleString affects the formatting of the date, not the timezone